### PR TITLE
[Review comments] Add some review comments, no need to merge this PR

### PR DESCRIPTION
**Pull request by jiahy0825**
[Review comments] Add some review comments, no need to merge this PR

### CODE REVIEW

File: CAPItools/pytools/cn/paddle/phi/api/include/tensor_utils/from_blob.rst [hunk 1]
```diff
@@ -24,5 +24,7 @@ paddle\phi\api\include\tensor_utils.h
 
 返回
 :::::::::::::::::::::
+#TODO：此处感觉写成下面这行比较好
+- **Tensor** - A Tensor object constructed from the buffer
```
[the remark this answers is not in the document]
Comment (Liyulingyue) [reply CAPItools/pytools/cn/paddle/phi/api/include/tensor_utils/from_blob.rst:28]: 这样对于当前的一些注释而言，会出现`@return bool`的内容，解析之后会变成 `返回\n:::::::::::::::::::::\nbool bool`

[the remark this answers is not in the document]
Comment (jiahy0825) [reply CAPItools/pytools/cn/paddle/phi/api/include/tensor_utils/from_blob.rst:28]: > 此处展示@return后面的内容，即只展示说明

查了下 doxygen 的示例，确实只展示说明即可～～

Ref link: https://www.midlane.top/wiki/display/utils/Doxygen
image

File: CAPItools/pytools/cn/paddle/phi/api/include/tensor/Tensor.rst [hunk 2]
```diff
@@ -254,6 +255,7 @@ Determine whether the tensor device is CustomDevice
 '''''''''''
 false
 
+# TODO： template <typename T>，这种模版类型的 case，需要有种方式展示出来
```
Comment (Liyulingyue) [inline CAPItools/pytools/cn/paddle/phi/api/include/tensor/Tensor.rst:258]: solved

**Comment by jiahy0825** [comment]
Review comments have been updated to a new PR.

**Pull request closed without merging**